k6/getReviewsMeta.js

```diff
@@ -0,0 +1,14 @@
+import http from 'k6/http';
```

Comment:
If you are trying for the last 10% of the data set for product_id, you should start around 900k. Not sure about other collections.

Reply:
Yeah I changed this later. Thank you!